src/components/paradox/results/personality/career/inline-jobs/auto.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from "prop-types";
+import {useEffect, useRef, useState} from "react";
+import InlineJobs from "./index";
+import style from "./style.scss";
+
+function InlineJobsAuto({jobs, jobSource, translate}) {
+  const ref = useRef(null);
+  const [count, setCount] = useState(0);
+
+  /** Adjusts number of jobs based on screen size */
+  const configureCount = () => {
+    const INLINE_JOB_WIDTH = 180;
```

Comment:
I think these variables should still just be regular type like inlineJobWidth.

Reply:
OK changed to camel case!